eureka-services/src/main/java/edu/emory/cci/aiw/cvrg/eureka/services/resource/ValueComparatorResource.java

```diff
@@ -47,13 +47,7 @@ public ValueComparatorResource(ValueComparatorDao valueCompDao) {
 	}
 	
 	@GET
-	@Path("/list")
-	public List<ValueComparator> getAll () {
-		return this.valueCompDao.getAll();
-	}
-	
-	@GET
-	@Path("/listasc")
+        @Path("")
```

Comment:
Jersey logs a warning on encountering an empty @Path. Just leave it out altogether.